docs/tutorials/multimodal/convert_data_to_coco_format.md

```diff
@@ -0,0 +1,169 @@
+# Convert Data to MSCOCO Format
+
+:label:`convert_data_to_coco_format`
+
+MSCOCO is one of the most popular datasets for object detection
+and its annotation format, usually referred to as the "MSCOCO format", has also been widely adopted.
+The "MSCOCO format" is a json structure that governs how labels and metadata are formatted for a dataset.
+We use MSCOCO format as the standard data format for training and inference in object detection tasks, and 
+require that all data related to object detection tasks should conform to the "MSCOCO format".  
+For details regarding MSCOCO dataset, please see this page.
+
+## How to prepare MSCOCO format
+### 1. Formatting folder Structure
+Under the MSCOCO format, the overall folder structure of a dataset should follow:
+```
+<dataset_dir>/
+    images/
+        <imagename0>.<ext>
+        <imagename1>.<ext>
+        <imagename2>.<ext>
+        ...
+    annotations/
+        train_labels.json
+        val_labels.json
+        test_labels.json
+        ...
+```
+
+### 2. Formatting ``*_labels.json``
+Below are the key names and value definitions inside ``*_labels.json``:
+
+```javascript
+{
+    "info": info,
+    "licenses": [license], 
+    "images": [image],  // list of all images in the dataset
+    "annotations": [annotation],  // list of all annotations in the dataset
+    "categories": [category]  // list of all categories
+}
+
+where:
+
+info = {
+    "year": int, 
+    "version": str, 
+    "description": str, 
+    "contributor": str, 
+    "url": str, 
+    "date_created": datetime,
+}
+
+license = {
+    "id": int, 
+    "name": str, 
+    "url": str,
+}
+
+image = {
+    "id": int, 
+    "width": int, 
+    "height": int, 
+    "file_name": str, 
+    "license": int,  // the id of the license
+    "date_captured": datetime,
+}
+
+category = {
+    "id": int, 
+    "name": str, 
+    "supercategory": str,
+}
+
+annotation = {
+    "id": int, 
+    "image_id": int,  // the id of the image that the annotation belongs to
+    "category_id": int,  // the id of the category that the annotation belongs to
+    "segmentation": RLE or [polygon], 
+    "area": float, 
+    "bbox": [x,y,width,height], 
+    "iscrowd": int,  // 0 or 1,
+}
+```
+For the sole purpose of running AutoMM, the fields ``"info"`` and ``"licenses"`` are optional. 
+``"images"``, ``"categories"``, and ``"annotations"`` are required.
+
+
+
+```json
+{
+  "info": {...},
+"licenses": [
```

Comment:
The indentation here seems not right, need 4 more spaces.

Reply:
Addressed in ce884dba67c46572c8b5c699980d4cfebf55cdd0